### DISCUSSION TITLE
Drag Reduction in Particle Simulations

### DISCUSSION THREAD

**Discussion by 19ap** [firemodels/fds#13122] [General]
Hello. 
I have a question regarding the solid particle simulations. I've modelled a series of tests to predict settling velocity and want to clarify some of the result interpretations. 
The model setups involve different numbers of point-injected particles (inserted from points forming a square horizontal surface) and varying grid resolutions. The particle diameter is 4×10^−6 m, DT is uniform and NOISE=F. 
Meshes with smaller cells and a higher count of particles show increased settling velocities (see the picture
![Pic1](https://github.com/firemodels/fds/assets/85547997/e9a0d9a6-1981-4867-897a-e29019d2b937)
), presumably due to drag reduction. The volume fraction in the case with the highest particle number (2.5 million) in the grid with the smallest cell (0.025 m) is 5.4×10^−6. However, the Technical Reference Guide states that "the drag reduction factor in Eq. 9.12 is only used when the local droplet volume fraction exceeds 1×10^−5". This implies that we should not observe the mentioned particle acceleration at this volume fraction. 
I might be missing something, so I would appreciate any comments or insights on this matter.
Thank you.

**Comment by mcgratta** [comment]
First, have you verified that a single particle settles at the appropriate terminal velocity given the drag coefficient that either you are FDS has specified. If so, I think that the drag reduction is not turned on if the density is less than the stated amount. Also, drag reduction only happens for liquid droplets. It is possible that you are seeing the effect of the surrounding gas being dragged downwards by the falling particles. Can you confirm if this is the case?

**Reply by 19ap** [reply]
Thank you for your reply, Kevin. In these cases, the drag coefficient is specified by FDS. The resulting settling velocity for 250 particles is within 5% (error) of that obtained using Stokes' law. 
It appears that it (surrounding gas dragged downwards by the falling particles) could be exactly the case, but I am uncertain about the appropriate method to verify this (just measure the air velocity?). Any suggestions would be greatly appreciated.

**Comment by mcgratta** [comment]
You can add a `DEVC` like
```
&DEVC ID='w', XYZ=..., QUANTITY='W-VELOCITY' /
```
and also add some slice files with `VECTOR=T`.

**Comment by 19ap** [comment]
I ran some simulations and would like to share the results for a case with 2.5 million particles and a cell size of 0.025 m:

1) Measurements by the point devices (as suggested by Kevin)
DEVCs were positioned in the middle of the particle cloud at heights corresponding to the particle injection plane (h_dev = 1.2 m) and every 0.2 m downward.
It is expected that a high concentration of particles, along with the momentum exchange between the falling particles and the fluid, would drag the surrounding air downward, creating a local flow field that enhances the settling velocity.
However, the curves indicate that the resulting air velocity is significantly lower than the settling velocity of the particles. 
![AIR W-Vel](https://github.com/firemodels/fds/assets/85547997/d7085655-56a0-421e-9fa1-4e29bea41407)
(The time frame when the particle cloud settles around a DEVC is marked in colour.)

2) 2D Slices
2D slices (w-velocity and velocity vector) placed in the middle of the domain (and particle cloud) support this suggestion.
![AIR_W-Vel_Slice](https://github.com/firemodels/fds/assets/85547997/21dfb821-6396-47d0-b57d-82b52b412169)
![AIR_Vel-Vector](https://github.com/firemodels/fds/assets/85547997/2a7ec2e9-17a0-407b-a028-4dd9bec75638)

These results lead me to believe that the primary factor behind the increased settling velocity could be particle clustering, which results in less drag per particle compared to isolated particles. For the model, we have the drag coefficient 𝐶𝐷=24/𝑅𝑒 (as 𝑅𝑒<1), and for stagnant air conditions, this value is relatively high.

Are there any recommendations for adjusting the drag coefficient when considering particle clustering? Is there something else we should consider, such as particles' aerodynamic interactions, which occur when the average particle spacing is less than 10 diameters (section "Drag Reduction")? Any insights or suggestions would be greatly appreciated.

**Comment by mcgratta** [comment]
There is a vast literature on this subject because there are many areas of engineering where particle or droplet motion is important. Simo Hostikka and co-workers at VTT and the U of Aalto, Finland, have looked at this issue extensively. FDS does not consider particle to particle interaction directly; that is, one particle does not "know" if there are particles nearby. FDS only knows the mass of particles per unit volume, and thus any empirical correlation must work with this parameter. We cannot (easily) account for one particle sitting in the wake of another. 

The good news is that if you find another approach that is based on quantities that FDS is already calculating, like mass per volume, one could easily test it because there are only a few lines of code dedicated to this sort of thing.

**Reply by 19ap** [reply]
Dear Kevin, thank you for your reply. I would like to clarify my understanding based on your insights.

In cases where there is a high particle count in a small volume (fine cell), the cumulative drag is reduced due to particle clustering. This reduction in drag could explain the observed increase in particle velocity.

Regarding the drag coefficient, since FDS does not directly account for particle-to-particle interactions, there isn't an easy way to account for these interactions directly. However, we can relate the effect of these interactions to a quantitative parameter such as the mass of particles per unit volume and develop a correlation for adjusting CD based on a series of simulations (as a start for drag coefficient determination).

Did I interpret your points correctly?

**Reply by mcgratta** [reply]
Do you see a greater downward velocity when the concentration of particles is relatively high?

**Reply by 19ap** [reply]
Yes, particles settle more quickly when their number increases, as shown in the picture from my initial post.

**Reply by mcgratta** [reply]
OK, so isn't this what we would expect, even if no special drag reduction algorithm is implemented?

**Reply by 19ap** [reply]
Yes, exactly, considering the particle clustering I've mentioned. 
The idea for adjusting CD is to predict the change in settling velocity for even larger quantities of injected particles without performing additional simulations.

**Reply by obscureed** [reply]
(I am not affiliated with NIST etc, so these are not necessarily expert comments):

Is it possible that your injected particles are actually representing a more significant flowrate than you expect? If you don't specify FLOW_RATE, or MASS_FLOW_RATE, or OPERATING_PRESSURE and K_FACTOR, then you may end up with the defaults of those last two parameters. These defaults are 1.0 and 1.0 respectively, with units such that you end up with FLOW_RATE = 1.0 L/min. (I think this is true; there might be differences between solid and liquid injections etc.) You need a clear distinction between the number of tracked particles and the number (or flowrate etc) of droplets that each particle represents in terms of fluid-particle coupling. As a first step, you could postprocess 'DROPLET VOLUME FRACTION' and check it is as expected.

Your initial question was specific about solid particles; @mcgratta has noted that drag reduction due to dense clouds of droplets apply to liquid droplets only. One easy step would be to rerun with very high DENSE_VOLUME_FRACTION, as the User Guide suggests, and hope to see the concentration effects disappear.

It might help to post a full input file.

**Reply by mcgratta** [reply]
I would guess that the particles are introduced via an `INIT` line rather than a nozzle. The parameters cited are related to nozzles or sprinklers. But @obscureed's last comment brings up an idea. You can replace your solid particles by liquid droplets with a very high evaporation temperature and then use the drag reduction algorithm. All this being said, I don't know if any of these algorithms are detailed enough for a study of dense particle interaction. It's not something that we focus on.

**Reply by obscureed** [reply]
I haven't worked out whether the drag reduction is the intended point of the simulations, or whether the question is why it is appearing unexpectedly/unintentionally/undesirably.

**Reply by 19ap** [reply]
Indeed, the particles are introduced via an INIT line. 
Thank you for your suggestions, I'll consider them for future simulations.
@obscureed The main purpose of the simulations was to observe how FDS deals with a dense particle cloud and what happens with particle sedimentation.

**Reply by obscureed** [reply]
When you change N_PARTICLES in your &INIT commands, do you also change MASS_PER_VOLUME? If you are simulating one tracked particle per real particle, then I think you should. If you don't. then you're getting another default of 1.0, this time in kg/m^3. (I'm still trying to tackle your question of why there appear to be concentration-dependent effects, even though by your calculations the dense volume fraction is not reached, at least on average.)

For increasingly large concentrations, you could think about using one tracked particle to represent multiple real particles. That way, you can generate higher concentrations without proportionate computational expense. Unfortunately, this makes it hard or impossible to set up with the correct or intended randomness. Random effects, from random injection locations, may explain some of your current observations. How many particles per cell are injected, on average, for your various combinations of particle number and cell resolution?

By the way, if you're approaching high enough concentrations that drag is reduced, I would expect that Brownian motion and coagulation  would also start to be relevant in reality. That would be even more true if your particles were slightly smaller, for example <1micron. Very small particles do not typically bounce off each other (unless they are colloidally stabilised in water etc, which is a different set of complications). These effects (and others, as @mcgratta has said while I was typing) have no chance of being included in simple particle-tracking methods.

**Comment by mcgratta** [comment]
One last comment. FDS does include empirical models of aerosol deposition via various mechanisms such as gravity and thermophoresis. The reason we rely on empirical models is because the physics of tiny particles within the boundary layer is very complex and cannot be adequately described using Lagrangian particles.
